README.md

```diff
@@ -42,36 +42,36 @@ We’ve optimized and hardened some of the most popular container images on Dock
 
```

Comment:
We also need to make changes to builder.lst and image.lst files and run image generator script